### ISSUE TITLE
[BUG] /photo_view-0.11.1/lib/src/core/photo_view_gesture_detector.dart:106:29: Error: The argument type 'PointerEvent' can't be assigned to the parameter type 'PointerDownEvent'. ERROR:  - 'PointerEvent' is from 'package:flutter/src/gestures/events.dart' ('/home/fagangjin/development/flutter/packages/flutter/lib/src/gestures/events.dart'). ERROR:  - 'PointerDownEvent' is from 'package:flutter/src/gestures/events.dart' ('/home/fagangjin/development/flutter/packages/flutter/lib/src/gestures/events.dart'). ERROR:     super.addAllowedPointer(event); ERROR:                             ^

### ISSUE THREAD

**Issue by luohao123** [fireslime/photo_view#423]
```
flutter doctor                                                                                      1 master!?
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, 2.3.0-13.0.pre.57, on Ubuntu 18.04.5 LTS 4.15.0-143-generic, locale en_HK.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.1)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 4.0)
[✓] VS Code (version 1.56.2)
[✓] Connected device (2 available)

• No issues found!

```

Seems the lastest broken.

**Comment by muhgad** [comment]
I have the same problem

**Comment by Raphicci** [comment]
This might be related to the new Flutter update 2.2, there is some breaking changes on Pointer events : https://flutter.dev/docs/release/breaking-changes/1-22-deprecations. Calling `dart fix` on the repo might fix it.

**Comment by Khaoz-Topsy** [comment]
I have the same issue, unfortunately @Raphicci the `dart fix` did not help. Going to downgrade Flutter to before 2.2 and try that

**Comment by Khaoz-Topsy** [comment]
You can get past this issue by using the beta channel for now. `flutter channel beta` & `flutter upgrade`

**Comment by renancaraujo** [comment]
Our releases officially support the beta channel only, but since this was supposed to be fixed by https://github.com/fireslime/photo_view/pull/420, you can depend directly on photoview's repository while it doesn't reach beta. 


If we release a version supporting this on pub now, our score goes down.

**Comment by dpedrinha** [comment]
OK, I didn't understand what needs to be done to fix this problem.
Do I need to change flutter to beta? Is that it?

Isn't the beta channel newer than the master?

Does it mean the flutter team rolling back this change?

**Comment by TheJulianJES** [comment]
Use the following for now:
```
  photo_view:
    git:
      url: git://github.com/fireslime/photo_view
      ref: 52685ab7beaf8a9107904bdb8a1590924015d016
```
https://github.com/fireslime/photo_view/pull/427 reverted https://github.com/fireslime/photo_view/pull/420

**Comment by renancaraujo** [comment]
This should be fixed on 0.12.0:  https://github.com/fireslime/photo_view/releases/tag/0.12.0

**Comment by renancaraujo** [comment]
Further discussions use #441.
